Sources/Document/Json/JSONObject.swift

```diff
@@ -150,6 +158,16 @@ public class JSONObject {
         self.context.push(operation: operation)
     }
 
+    private func setValue<T: YorkieCountable>(key: String, value: CRDTCounter<T>) {
+        self.setAndRegister(key: key, value: value)
+
+        let operation = SetOperation(key: key,
+                                     value: value.deepcopy(),
+                                     parentCreatedAt: self.target.createdAt,
+                                     executedAt: self.context.issueTimeTicket())
+        self.context.push(operation: operation)
+    }
+
```

Comment:
I'm sorry. I made duplicated codes. It can be the following code.
```
private func setValue(key: String, value: CRDTElement) {
        self.setAndRegister(key: key, value: value)

        let operation = SetOperation(key: key,
                                     value: value.deepcopy(),
                                     parentCreatedAt: self.target.createdAt,
                                     executedAt: self.context.issueTimeTicket())
        self.context.push(operation: operation)
    }
```
And, the following codes can be removed.
```
private func setValue(key: String, value: CRDTObject) {
        self.setAndRegister(key: key, value: value)
        let operation = SetOperation(key: key,
                                     value: value.deepcopy(),
                                     parentCreatedAt: self.target.createdAt,
                                     executedAt: self.context.issueTimeTicket())
        self.context.push(operation: operation)
    }
    private func setValue(key: String, value: CRDTArray) {
        self.setAndRegister(key: key, value: value)
        let operation = SetOperation(key: key,
                                     value: value.deepcopy(),
                                     parentCreatedAt: self.target.createdAt,
                                     executedAt: self.context.issueTimeTicket())
        self.context.push(operation: operation)
    }

    private func setValue<T: YorkieCountable>(key: String, value: CRDTCounter<T>) {
        self.setAndRegister(key: key, value: value)

        let operation = SetOperation(key: key,
                                     value: value.deepcopy(),
                                     parentCreatedAt: self.target.createdAt,
                                     executedAt: self.context.issueTimeTicket())
        self.context.push(operation: operation)
    }

```

Reply:
Done!